web/components/Canvas/Board/Cards/Cards.module.scss

```diff
@@ -4,13 +4,14 @@
   .card {
     position: relative;
     width: 100px;
-    margin-left: -50px;
+    margin-left: 0px;

```

Comment:
this makes the cards not overlap at the bottom of the screen anymore, but now they don't spill off the side of the suggestion dialog